zhaquirks/centralite/3321S.py

```diff
@@ -0,0 +1,95 @@
+from zigpy.profiles import PROFILES, zha
+from zigpy.zcl.clusters.general import Basic, Identify,\
+    PollControl, Ota
+from zigpy.zcl.clusters.security import IasZone
+from zhaquirks.centralite import PowerConfigurationCluster
+from zhaquirks.smartthings import SmartThingsAccelCluster
+from zigpy.zcl.clusters.measurement import TemperatureMeasurement
+from zigpy.quirks import CustomDevice
+
+
+DIAGNOSTICS_CLUSTER_ID = 0x0B05  # decimal = 2821
+MANUFACTURER_SPECIFIC_PROFILE_ID = 0xC2DF  # decimal = 49887
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0xFC0F  # decimal = 64527
+
+class CentraLite3321S(CustomDevice):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1 profile=260 device_type=1026
+        #  device_version=0
+        #  input_clusters=[0, 1, 3, 1026, 1280, 32, 2821, 64514]
+        #  output_clusters=[25]>
+        1: {
+            'profile_id': zha.PROFILE_ID,
```

Comment:
we match on man / model now so add them here

Reply:
Done